src/GslCore/PrimerCreation.fs

```diff
@@ -181,7 +181,14 @@ let tuneTails
     let fullTemplate = DnaOps.concat [ rev.body.RevComp() ; middleDNA ; fwd.body ]
 
     /// Target Tm for middle annealing part.  Cheat if it's a linker and we just want to keep this part (ideally) full length
-    let annealTarget = match firmMiddle with | Some(x) -> x | None -> dp.seamlessOverlapTm
+    let annealTarget = 
+            match firmMiddle with 

```

Comment:
Please normalize indentation to four spaces per level.

Reply:
Done.

Reply:
This section is still 8 spaces per indent.

Reply:
apologies,  184G7<<